module/apmstreadwayamqp/utils.go

```diff
@@ -0,0 +1,64 @@
+package apmstreadwayamqp
+
+import (
+	"fmt"
+	"go.elastic.co/apm/module/apmhttp/v2"
+	"go.elastic.co/apm/v2"
+	"strings"
+)
+
+var (
+	elasticTraceparentHeader = strings.ToLower(apmhttp.ElasticTraceparentHeader)
+	w3cTraceparentHeader     = strings.ToLower(apmhttp.W3CTraceparentHeader)
+	tracestateHeader         = strings.ToLower(apmhttp.TracestateHeader)
+	apmStreadwayAmqpPrefix   = "apmstreadwayamqp_transaction_"
```

Comment:
Why are you adding a prefix? This means that distributed tracing will not work when producers/consumers are written in different languages, use different clients, or use different tracers (e.g. OTel + Elastic APM). Can we just use the standard "traceparent" and "tracestate" names?

Reply:
I see, thanks for elaborating.

> to distribute a trace through a message

Users can still send trace events through messages - they would be in the message body. What we might break is having two separate instrumentation libraries (e.g. OTel + Elastic APM) enabled, which both want to set `traceparent`. Users will just have to choose one or the other.

I would prefer to always set the headers, and document this behaviour. Otherwise if an error were returned, then an application might start failing just because two instrumentation libraries have been used (perhaps inadvertently). While distributed tracing might be broken, I think that's preferable to breaking the user's application.

Reply:
> I would prefer to always set the headers, and document this behaviour. Otherwise if an error were returned, then an application might start failing just because two instrumentation libraries have been used (perhaps inadvertently). While distributed tracing might be broken, I think that's preferable to breaking the user's application. 

That's reasonable, I agree. 👍